examples/streams.rs

```diff
@@ -235,21 +256,7 @@ fn read_group_records(client: &redis::Client, slowness: u8) -> RedisResult<()> {
         )
         .expect("records");
 
-    for StreamKey { key, ids } in srr.keys {
```

Comment:
Arbitrarily moved this code block.  The placement doesn't matter very much since it's an example, but it lines up somewhat nicely with the fact that `XACK` expects a group of IDs tied to a particular stream key (as opposed to being able to acknowledge multiple sets of IDs across multiple keys).